### PR TITLE
Dashboard Prettified

### PR DESCRIPTION
**Pull request by vipj**
- Prettified Dashboard page and added DynamoDB functionality
- Fixed small issue with login function
- Added forgot password flow back to codebase

### CODE REVIEW

File: src/components/CompleteApplicationButton/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      );
+    } else if (this.state.status === "enabled") {
+      return (
+        <Button className={styles.button} href={"/application"}>
+          {this.state.text}
+        </Button>
+      );
+    } else {
+      return null;
+    }
```
Comment (kyleyarwood) [inline src/components/CompleteApplicationButton/index.js:31]: Can the status of this ever be anything but "enabled" or "disabled"? If not, I'm wondering if we'd be able to get rid of this case and make the above one an else.

File: src/components/DashboardSidebarButtons/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import Button from "react-bootstrap/Button";
+
+import styles from "./styles.module.css";
+
+export default class DashboardSidebarButtons extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      text: props.text,
```
Comment (kyleyarwood) [inline src/components/DashboardSidebarButtons/index.js:11]: Is this property needed for this button?

File: src/views/DashboardPage/index.js [hunk 3]
```diff
@@ -6,41 +6,99 @@ import Button from "react-bootstrap/Button";
...
+};
+
+const tableName = process.env.REACT_APP_DYNAMO_DB_TABLE;
+
 const userPool = new CognitoUserPool(poolData);
 var cognitoUser = userPool.getCurrentUser();
+var dynamoDB = new AWS.DynamoDB(dynamoDbData);
 var isLoggedIn = false;
 
-window.onload = function () {
```
Comment (kyleyarwood) [inline src/views/DashboardPage/index.js:19]: I'm not super well-versed in React haha, so I'm kinda curious, what was this doing and why were we able to get rid of it?

File: src/components/DashboardSidebarButtons/index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      );
+    } else if (this.state.type === "application") {
+      return (
+        <Button href={"/application"} variant="link" className={styles.button} block>
+          <h1 className={styles.text}>
+            Application
+          </h1>
+        </Button>
+      );
+    } else if (this.state.type === "logout") {
```
Comment (dunja-tomic) [inline src/components/DashboardSidebarButtons/index.js:33]: A logout function should probably be called here?

File: src/views/ForgotPasswordPageChange/index.js [hunk 5]
```diff
@@ -1,34 +1,76 @@
...
+      });
+    } else {
+      this.setState({ err: "noMatch" });
+    }  
   };
+  
 
   displayErrors = () => {
-    if (this.state.err) {
+    if (this.state.err === "noCode") {
```
Comment (dunja-tomic) [inline src/views/ForgotPasswordPageChange/index.js:67]: I like this, nice!

**Pull request closed without merging**